pkgs/by-name/hy/hyprland-qtutils/package.nix

```
@@ -0,0 +1,74 @@
+{
+  lib,
+  stdenv,
+  aquamarine,
+  cairo,
+  cmake,
+  fetchFromGitHub,
+  hyprcursor,
+  hyprgraphics,
+  hyprlang,
+  hyprutils,
+  libinput,
+  pango,
+  pciutils,
+  pixman,
+  pkg-config,
+  qt6,
+  re2,
+  wayland-protocols,
+  xgboost,
+  xorg,
+}:
+let
+  inherit (lib.sources) cleanSource cleanSourceWith;
+  inherit (lib.strings) hasSuffix makeBinPath;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hyprland-qtutils";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "hyprwm";
+    repo = "hyprland-qtutils";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-dj9dpVwpyTmUyVu4jtaIU39bHgVkoZjv6cgYfWyHc9E=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    aquamarine
+    cairo
+    hyprcursor
+    hyprgraphics
+    hyprlang
+    hyprutils
+    libinput
```

```suggestion
    libgbm
    libinput
```

Do you just need to add this?

---

These dependencies are not even needed. Something went horribly wrong (I'm guessing hash reuse from Hyprland that confused Nix). I'll push a fix soon.